### PR TITLE
admin navagation fragment

### PR DESCRIPTION
**Pull request by Wumbo12347**
added xml,  and fragment class for admin navigation page

### CODE REVIEW

File: napkinapp/app/src/main/res/layout/admin_navagation.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="16dp"
+    android:gravity="top"
+    android:background="#FAF5FF"> <!-- Light purple background -->
```
Comment (Sherali5543) [inline napkinapp/app/src/main/res/layout/admin_navagation.xml:9]: why is there a background

File: napkinapp/app/src/main/java/com/example/napkinapp/fragments/adminmenu/AdminNavagationFragment.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package com.example.napkinapp.fragments.adminmenu;
```
Comment (Sherali5543) [inline napkinapp/app/src/main/java/com/example/napkinapp/fragments/adminmenu/AdminNavagationFragment.java:1]: file should be in it's own folder within fragments folder

**Review by Sherali5543 — CHANGES_REQUESTED** [review]
Will make a pull later today to review ui

**Review by davidurschel — APPROVED** [review]
seems good UI

**Pull request merged**